x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -313,6 +313,14 @@ function ObservabilityActions({
   );
 }
 
+const FIELDS_WITHOUT_CELL_ACTIONS = [
```

Comment:
@claudia-correia this PR resolves the `TODO` from https://github.com/elastic/kibana/pull/116446/files#L149 by moving the constant here, and providing the value via the `disabledCellActions` prop

Reply:
Nice improvement! This makes the code more generic and allows different actions for the same fields across the product.